### ISSUE TITLE
Term Request: Railroad-track corneal endothelial bands

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11752]
**Preferred term label:**
Railroad-track corneal endothelial bands

**Synonyms**
* Endothelial railroad-track lines
* Parallel linear endothelial bands
* Scalloped endothelial ridges

**Definition (free text, please give PubMed ID)**
Parallel, linear, gray-white endothelial ridges or bands with irregular scalloped borders and white flaky margins, extending across Descemet membrane and the endothelial surface, characteristic of endothelial-to-epithelial cell transition (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Abnormal corneal endothelium morphology (HP:0011488)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Posterior polymorphous corneal dystrophy (PPCD) (ORPHA:98973; PPCD1 OMIM:122000, PPCD3 OMIM:609141, PPCD4 OMIM:618031)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
